README.md

```diff
@@ -1,68 +1,69 @@
-[![PyPI version](https://badge.fury.io/py/aiida-core.svg)](https://badge.fury.io/py/aiida-core)
-
-
-AiiDA
-=====
-
-**Automated Interactive Infrastructure and Database for Computational
-Science**
-
-AiiDA is a sophisticated framework designed from scratch to be a
-flexible and scalable infrastructure for computational science. Being
-able to store the full data provenance of each simulation, and based on
-a tailored database solution built for efficient data mining
-implementations, AiiDA gives the user the ability to interact seamlessly
-with any number of HPC machines and codes thanks to its flexible plugin
-interface, together with a powerful workflow engine for the automation
-of simulations.
-
-The official homepage is at <http://www.aiida.net>
-
-The code is hosted on GitHub at
-<https://github.com/aiidateam/aiida-core>
-
-The documentation is hosted on Read The Docs at:
-
--   [stable version](http://aiida-core.readthedocs.io/en/stable)
--   [develop version](http://aiida-core.readthedocs.io/en/latest)
-
-Which branch should you use?
-----------------------------
-
--   **Users**: the stable version of the code is in the [master
-    branch](https://github.com/aiidateam/aiida-core/tree/master)
--   **Developers**: the procedure to contribute through pull-requests
-    can be found in the
-    [wiki](https://github.com/aiidateam/aiida-core/wiki/Contributing-guide-and-branch-descriptions)
-
-How to cite AiiDA
------------------
-
-If you use AiiDA in your research, please consider citing the following
-work:
+# <img src="docs/source/images/AiiDA_transparent_logo.png" alt="AiiDA" width="200"/>
+
+AiiDA (www.aiida.net) is a python framework that aims to help researchers with managing complex workflows and making them fully reproducible.
+
+|    | |
+|-----|----------------------------------------------------------------------------|
+|Latest release| [![PyPI version](https://badge.fury.io/py/aiida-core.svg)](https://badge.fury.io/py/aiida-core) [![conda-forge](https://img.shields.io/conda/vn/conda-forge/aiida-core.svg?style=flat)](https://anaconda.org/conda-forge/aiida-core) [![PyPI pyversions](https://img.shields.io/pypi/pyversions/aiida-core.svg)](https://pypi.python.org/pypi/aiida-core/) |
+|Getting help| [![Docs status](https://readthedocs.org/projects/aiida-core/badge)](http://aiida-core.readthedocs.io/) [![Google Group](https://img.shields.io/badge/-Google%20Group-lightgrey.svg)](https://groups.google.com/forum/#!forum/aiidausers)
+|Build status| [![Build Status](https://travis-ci.org/aiidateam/aiida-core.svg?branch=develop)](https://travis-ci.org/aiidateam/aiida-core) [![Coverage Status](https://coveralls.io/repos/github/aiidateam/aiida-core/badge.svg?branch=develop)](https://coveralls.io/github/aiidateam/aiida-core?branch=develop) |
+|License| [![License](https://img.shields.io/github/license/aiidateam/aiida-core.svg)](https://github.com/aiidateam/aiida-core/blob/develop/LICENSE.txt)|
```

Comment:
License shows as not identifiable by Github. Do we know why and can we fix it?

Reply:
I think it has to do with the addition at the bottom of the license.
This PR moves the addition from there to the README